### DISCUSSION TITLE
4.6.0

### DISCUSSION THREAD

**Discussion by ywmoyue** [ywmoyue/biliuwp-lite#80] [General]
#### CHANGED 
- 视频评论区支持查看图片
- 更新WinUI版本至2.8.3
  - ⚠最低支持操作系统版本变更：Windows 10 version 1903
  - 代码来自https://github.com/xiaoyaocz/biliuwp-lite/pull/225 ,感谢@iloxmn
- 优化简繁体转换实现方式
  - 方案来自https://github.com/ywmoyue/biliuwp-lite/discussions/52  ，感谢@xiaoyaocz
#### FIXED
- 修复评论区无IP属地显示
- 修复查询用户资料及投稿509错误

<hr /><em>This discussion was created from the release <a href='https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.6.0'>4.6.0</a>.</em>

**Comment by TPAXcc** [comment]
你好快啊~
